src/docs/sdk/event-payloads/transaction.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Transaction Payloads
+---
+
+Transactions are used to send tracing events to Sentry.
+
+Transactions must be wrapped in an [Envelope](/sdk/envelopes) and therefore also be sent to the Envelope endpoint.
+
+<Alert title="Note on backwards compatibility" level="info">
+
+Envelopes are a new format and therefore only work on Sentry >= 10. More infromation can be found on the [Envelope](../envelopes) page.
+</Alert>
+
+## Anatomy
+
+A Transaction is basically a [Span](/sdk/event-payloads/span) combined with an [Event](/sdk/event-payloads/). When using 
+tracing with our SDKs you usually create a Span tree, the root node and therefore the whole tree is considered to be the Transaction.
+So technically a Transaction is just a Span. A Transaction must also have a `contexts.trace` (which contains some data of the [Span](/sdk/event-payloads/span)) and some other
+properties that will be covered in the next section.
+
+Transactions is an [Events](/sdk/event-payloads/) which will be enriched with data from the [Span](/sdk/event-payloads/span).
+We are only going to list here whats important for a Transaction.
+
+import "./properties/type.mdx"
+
+import "./properties/start_timestamp.mdx"
+
+import "./properties/timestamp.mdx"
+
+### contexts.trace
+
+A Transaction has to have a specifc `contexts.trace` entry that contains data from the [Span](/sdk/event-payloads/span).
+
+import "./properties/span_id.mdx"
+
+import "./properties/parent_span_id.mdx"
+
+import "./properties/trace_id.mdx"
+
+import "./properties/op.mdx"
+
+import "./properties/description.mdx"
+
+## Examples
+
+```json
+{
+    "contexts": {
+        "trace": {
+            "op": "navigation",
+            "description": "User clicked on <Link />",
+            "trace_id": "743ad8bbfdd84e99bc38b4729e2864de",
+            "span_id": "a0cfbde2bdff3adc"
+        }
```

Comment:
Need to review/align this with code in Relay/SDKs.

---

I looked at an existing transaction sent from the JS SDK and it has an additional `"type": "trace"`.

```json
    "contexts": {
        "trace": {
            "description": "[redacted]",
            "op": "[redacted]",
            "type": "trace",
            "trace_id": "f0eeddd8e915418b953ff969df526b2d",
            "span_id": "96f3b555b5c7ad45"
        }
    },
```

And looked at Relay: https://github.com/getsentry/relay/blob/53a260e6de70d1fec496aaf8c9348f0c5a17e5d8/relay-general/src/protocol/contexts.rs#L360-L383

- Relay ignores the `type` key (we should fix it in the SDK)
- Relay ignores the `description` (goes to `other` field)
- Relay knows of a `status` which is not present here
- Relay knows of a `parent_span_id`, also not here